libs/langchain/langchain/llms/llamacpp.py

```diff
@@ -246,13 +249,30 @@ def _call(
                 llm = LlamaCpp(model_path="/path/to/local/llama/model.bin")
                 llm("This is a prompt.")
         """
+        if self.grammar_path:
+            try:
+                from llama_cpp import LlamaGrammar
```

Comment:
This line can also fail if llama_cpp does not implement LlamaGrammar. Then it will raise import error so it would be better to let the user know that this feature is missing in this version.

Reply:
I've added more verbose description

Reply:
I added ImportError